src/RegisterHandler.php

```diff
@@ -259,7 +259,7 @@ protected function getCredentialCreationOptions(
             $this->getUserEntity($store->getMember()),
             random_bytes(32),
             [new PublicKeyCredentialParameters('public-key', Algorithms::COSE_ALGORITHM_ES256)],
-            40000
+            $this->getAuthenticatorSelectionCriteria()
```

Comment:
```suggestion
            $this->getAuthenticatorSelectionCriteria(),
            timeout: 4000
```
We used to pass this in here as well, so lets keep doing so.

Reply:
Updated

Reply:
v4.4 doesn't have `$timeout` parameter in constructor.